package.json

```diff
@@ -18,13 +15,12 @@
     "deploy": "npm run dateData && npm run deploySite",
     "deploySite": "npm run buildSite && gh-pages -a -d dist/ -e .",
     "lint": "eslint .",
-    "postinstall": "node initSSL.js",
-    "start": "NODE_OPTIONS='--insecure-http-parser' node cli.js",
+    "start": "node cli.js",
     "test": "jest",
     "timeseries": "node timeseries.js",
```

Comment:
What's curious is that we never passed that option to `timerseries` -- looks like a bug.

Reply:
Fixed in master anyway, stoked to nuke it completely :)